src/app/app.py

```diff
@@ -111,9 +111,9 @@ async def get_schema(survey_id: str, version: str) -> dict:
 
 
 @app.get("/v1/schema_metadata", response_model=list[ReturnedSchemaMetadata])
-async def get_schemas_metadata(survey_id: str = '') -> list[ReturnedSchemaMetadata]:
+async def get_schemas_metadata(survey_id: str = "") -> list[ReturnedSchemaMetadata]:
     """Retrieve the metadata for all the schemas that have a given survey_id."""
-    if survey_id == '':
+    if survey_id == "":
         return exception_throw.throw_400_incorrect_schema_key_exception()
```

Comment:
I have forgotten to add a logger.error() here. May be best to keep exception and log error paired